### PR TITLE
Fixed which blocks can be harvested with what type of pickaxe

### PR DESCRIPTION
**Pull request by Bond-009**
Source: http://minecraft.gamepedia.com/Pickaxe#Usage

### CODE REVIEW

File: src/Items/ItemPickaxe.h [hunk 1]
```diff
@@ -60,25 +60,39 @@ class cItemPickaxeHandler :
 
 			case E_BLOCK_ANVIL:
 			case E_BLOCK_BONE_BLOCK:
+			case E_BLOCK_BLOCK_OF_COAL:
+			case E_BLOCK_BLOCK_OF_REDSTONE:
+			case E_ITEM_BREWING_STAND:
```
Comment (madmaxoft) [inline src/Items/ItemPickaxe.h]: `ITEM`? Shouldn't this be a `BLOCK`? (Lucky for us Travis caught this)

File: src/Items/ItemPickaxe.h [hunk 2]
```diff
@@ -97,6 +113,7 @@ class cItemPickaxeHandler :
 			case E_BLOCK_STONE_BRICK_STAIRS:
 			case E_BLOCK_STONE_PRESSURE_PLATE:
 			case E_BLOCK_STONE_SLAB:
+
```
Comment (madmaxoft) [inline src/Items/ItemPickaxe.h]: Please remove this extra empty line.

File: src/Items/ItemPickaxe.h (hunk 1 above)
Comment (Bond-009) [inline src/Items/ItemPickaxe.h]: My bad, it's fixed now.

**Comment by madmaxoft** [comment]
Thanks

**Pull request merged**